pkg/agent/util/powershell/powershell_windows.go

```diff
@@ -27,7 +27,7 @@ func RunCommand(cmd string) (string, error) {
 	// https://stackoverflow.com/questions/19282870/how-can-i-use-try-catch-and-get-my-script-to-stop-if-theres-an-error/19285405
 	psCmd := exec.Command("powershell.exe", "-NoLogo", "-NoProfile", "-NonInteractive", "-Command",
 		fmt.Sprintf(`$ErrorActionPreference="Stop";try {%s} catch {Write-Host $_;os.Exit(1)}`, cmd)) // #nosec G204
-	stdout, err := psCmd.Output()
+	stdout, err := psCmd.CombinedOutput()
```

Comment:
When I tested this powershell function with CombinedOutput support, I got this error: os.Exit : The term ‘os.Exit’ is not recognized as the name of a cmdlet, function, script file, or operable program.
	Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
	At line:1 char:211
	+ ... .2.6 -InternalPort 80 -Protocol tcp} catch {Write-Host $_;os.Exit(1)}
	+                                                               ~~~~~~~
	    + CategoryInfo          : ObjectNotFound: (os.Exit:String) [], ParentContainsErrorRecordException
	    + FullyQualifiedErrorId : CommandNotFoundException
	
	’, exit status 1

Reply:
Which command? `CombinedOutput` is used to output error info, which `Output` ignores error info.

Reply:
Yes, I tested a netnatstaticmapping function that includes the runCommand(cmd). When I replaced Output with CombinedOutput, I got this error. I think this issue has always been there, previously we didn't get it because psCmd.Output() ignore this error.

Reply:
e.g.
try { Get-Process} catch { Write-Host $_; os.Exit(1) }
pass
try { Get-Process error} catch { Write-Host $_; os.Exit(1) }
Cannot find a process with the name "error". Verify the process name and call the cmdlet again. 
os.Exit : The term 'os.Exit' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path  
is correct and try again.

My concern is after you enable CombinedOutput in RunCommand function, all error commands may bring this same error log.

Reply:
Do you mean that we need to add some code to ignore the error which was ignored by `Output` previously?

Reply:
@tnqn After syncing with @wenyingd @XinShuYang , the error output is caused by `os.Exit(1)` since its not powershell command. We can fix it by changing  `os.Exit(1)`  to `Exit(1)`